docs/FeatureManagement/FeatureFlag.v1.1.0.schema.json

```diff
@@ -56,6 +56,16 @@
       "description": "The declaration of conditions used to dynamically enable features.",
       "required": [],
       "properties": {
+        "requirement_type": {
+          "$id": "#/properties/conditions/properties/requirement_type",
+          "type": "string",
+          "title": "Requirement Type",
+          "description": "Determines whether any or all registered feature filters must be enabled for the feature to be considered enabled.",
+          "examples": [
+            "Any",
+            "All"
+          ]
```

Comment:
What's the default?

Reply:
Default functionality is Any. However here, I was treating the default as excluding the field (As FeatureManagement decides what the "default" is), but perhaps default means "when the field is included, what is the default". 

What do you think?

Reply:
We can include a [default annotation](https://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.7.7.1.1) just for informational purposes.